neqo-transport/src/crypto.rs

```diff
@@ -1034,15 +1043,27 @@ impl CryptoStates {
             Role::Server => (SERVER_INITIAL_LABEL, CLIENT_INITIAL_LABEL),
         };
 
+        let min_pn = if randomize_first_pn {
+            let r = random::<4>();
+            let pn = packet::Number::from;
+            // A random starting packet number. 5 uniformly random bits.
+            // Then maybe add up to 256 to occasionally nudge into a second byte for
+            // both varint and packet number encodings, heavily weighted to small values.
+            // And a small offset to ensure that the value is always non-zero.
+            (pn(r[0]) & 0x1f) + (pn(r[1]) + pn(r[2]) + pn(r[3])).saturating_sub(512) + 1
```

Comment:
Why not just the below?

```suggestion
            pn(r[0].min(r[1]).min(r[2]).min(r[3])) + 1
```

- Easy to reason about.
- Always above `0`.
- 8 bits long in 6.25% (0.5^4) of the case, i.e. 2 byte in variable length encoding.
- Add or remove `min`s to increase or decrease probability for 2 byte long variable length encoding.